libs/auth-api-lib/src/lib/delegations/delegations-outgoing.service.ts

```diff
@@ -103,7 +105,34 @@ export class DelegationsOutgoingService {
       ],
     })
 
-    return delegations.map((d) => d.toDTO())
+    const delegationTypesDelegations = await this.delegationModel.findAll({
+      where: {
+        fromNationalId: user.nationalId,
+      },
+      include: [
+        {
+          model: DelegationDelegationType,
+          where: {
+            delegationTypeId: AuthDelegationType.GeneralMandate,
+            validTo: {
+              [Op.or]: {
+                [Op.gt]: new Date(),
```

Comment:
We talked earlier about validTo being just the day. So use `gte` and `startOfDay`